src/configurations/nf/synapseConfigs/tools.ts

```diff
@@ -65,22 +65,277 @@ const tools: HomeExploreConfig = {
       name: 'Tools',
       cardConfiguration: toolsCardConfiguration,
       facetAliases,
-      searchConfiguration: {
-        searchable: [
-          'name',
-          'summary',
-          'studyName',
-          'fundingAgency',
-          'contact',
-          'type',
-          'subtype',
-          'diseaseFocus',
-          'manifestation',
-        ],
-      },
-
     },
   },
 }
 
+export const toolDetailsPageConfig: DetailsPageProps = {
+  sql: toolsSql,
+  sqlOperator: '=',
+  tabLayout: [
+    {
+      title: "Details",
+      synapseConfigArray: [
+        {
+          name: 'UserCardListRotate',
+          title: 'Tool Origin',
+          subtitle: 'Investigator',
+          props: {
+            sql: investigatorSql,
+            count: 1,
+            size: SynapseConstants.MEDIUM_USER_CARD,
+            useQueryResultUserData: true,
+            sqlOperator: '=',
+          },
+          tableSqlKeys: ['resourceId'],
+          columnName: 'resourceId',
+        },
+        {
+          name: 'CardContainerLogic',
+          title: 'Development Publication',
+          props: {
+            limit: 3,
+            facetAliases,
+            sql: developmentPublicationSql,
+            secondaryLabelLimit: 4,
+            ...publicationsV2CardConfiguration,
+          },
+          tableSqlKeys: ['resourceId'],
+          columnName: 'resourceId'
+        },
+        {
+          name: 'SubsectionRowRenderer',
```

Comment:
Many instances of the SubsectionRowRenderer, which runs a query (supports URL search param based query filter), that ultimately may return a row to be rendered.  Looks at the column type to determine how to render the value.  Column name is used as the section header.